src/styles/addbook.css

```diff
@@ -0,0 +1,67 @@
+form {
```

Comment:
- [ ] I think we could add some horizontal padding to the form so it's centered just like the template from the design. Let's try to use the same padding for the navbar, books-cards, and these sections. This will generate the feeling that the elements are all centered and have the same width. 

Yours:
![image](https://user-images.githubusercontent.com/79114233/142738626-0c177546-0c40-40c7-93eb-d035316a0f98.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/142738678-2f621b16-3d25-4f65-bb89-cfee0cfc78f8.png)